<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>765</width>
    <height>681</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lista do Karaokê</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>karaoke.png</normaloff>karaoke.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(67, 23, 23);
background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Keraleeyam</family>
          <pointsize>10</pointsize>
          <weight>9</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 italic 10pt &quot;Keraleeyam&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; color:#e9b96e;&quot;&gt;KARAOKE RAUL ROCK BAR &amp;amp; CAFÉ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 87, 83);
border-color: rgba(191, 64, 64, 0);
Qframe{
border-radius: 10px;

}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>4</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(136, 138, 133);</string>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Insira um nome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Adicionar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Atualizar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(115, 210, 22);</string>
           </property>
           <property name="text">
            <string>  HORÁRIO   </string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>22</horstretch>
             <verstretch>4</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>41</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>27</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(138, 226, 52);
border-color: rgb(136, 138, 133);</string>
           </property>
           <property name="text">
            <string> 00:00 </string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">alternate-background-color: rgb(245, 121, 0);
gridline-color: rgb(136, 138, 133);
border-color: rgb(255, 255, 255);
selection-color: rgb(92, 53, 102);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>41</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>195</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>24</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>32</number>
        </attribute>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nome</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Falta...</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Obs.:</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>200</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(138, 226, 52);</string>
       </property>
       <property name="text">
        <string>Cantou!</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>200</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>200</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 41, 41);</string>
       </property>
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>200</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Histórico</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>231</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>logokaraoke.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>570</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(26, 103, 252)</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;a href=&quot;https://www.linkedin.com/in/welton-moura-23a897230&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#8ae234;&quot;&gt;CREATED BY WELTON MOURA&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>pushButton_5</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
